packages/core/src/App/Containers/AccountSwitcher/account-switcher.jsx

```diff
@@ -426,7 +426,12 @@ class AccountSwitcher extends React.Component {
                         {this.remaining_real_accounts.map((account, index) => (
                             <div key={index} className='acc-switcher__new-account'>
                                 <Icon icon='IcDeriv' size={24} />
-                                <span className='acc-switcher__new-account-text'>{getAccountTitle(account)}</span>
+                                <span className='acc-switcher__new-account-text'>
+                                    {getAccountTitle(account, {
+                                        is_isle_of_man_residence: this.props.is_im,
```

Comment:
can pass residence here instead of is iom or is_be, and handle the logic in getaccount title

Reply:
Thanks, changed accordingly